src/sql-parser/src/parser.rs

```diff
@@ -7447,45 +7447,43 @@ impl<'a> Parser<'a> {
             // Parse: `REPLAN INDEX name`
             Explainee::ReplanIndex(self.parse_raw_name()?)
         } else {
-            let broken = self.parse_keyword(BROKEN);
```

Comment:
Will this break any existing user views or queries? If we are not certain, we should keep this around as a no-op for a while.

Reply:
It is behind `enable_explain_broken` which is only enabled on 3 staging envs, and is never recorded to the catalog, so I think we should be fine?

Reply:
Great, parser change lgtm then.